lib/npg_pipeline/archive/file/logs.pm

```diff
@@ -0,0 +1,128 @@
+package npg_pipeline::archive::file::logs;
+
+use Moose;
+use Carp;
+use English qw{-no_match_vars};
+
+extends qw{npg_pipeline::base};
+
+our $VERSION = '0';
+
+has 'irods_root' => ( isa => 'Str',
+                      is  => 'rw',
+                    );
+
+sub submit_to_lsf {
+  my ($self, $arg_refs) = @_;
+  my $job_sub = $self->_generate_bsub_command($arg_refs);
+  my $job_id = $self->submit_bsub_command($job_sub);;
+  return ($job_id);
+}
+
+sub _generate_bsub_command {
+  my ($self, $arg_refs) = @_;
+
+  my $irodsinstance = $self->function_list() =~ /gclp/ismx ? q(gclp) : q();
+  my $id_run = $self->id_run();
+
+  my $required_job_completion = $arg_refs->{'required_job_completion'};
+  my $timestamp = $self->timestamp();
+  my $archive_script = q{irods_log_loader.pl};
```

Comment:
> We already have: /software/npg/bin/irods_bam_meta_updater.pl 
> /software/npg/bin/irods_consent_withdrawn_auto.pl 
> /software/npg/bin/irods_ebi_submission_data.pl 
> /software/npg/bin/irods_pacbio_loader.pl 
> /software/npg/bin/irods_bam_loader.pl 
> /software/npg/bin/irods_loader.pl 
> /software/npg/bin/irods_interop_loader.pl 
> /software/npg/bin/irods_pacbio_monitor.pl 
> /software/npg/bin/irods_bam_list_deletion.pl 
> /software/npg/bin/irods_pacbio_ebi_submission_data.pl 
> /software/npg/bin/irods_pacbio_meta_updater.pl 
> /software/npg/bin/irods_bam_deletion.pl Unless we rename everything 
> all at once I think having an inconsistent naming scheme would cause 
> more problems than it solves.
> … <#>
> -- The Wellcome Trust Sanger Institute is operated by Genome Research 
> Limited, a charity registered in England with number 1021457 and a 
> company registered in England with number 2742969, whose registered 
> office is 215 Euston Road, London, NW1 2BE.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wtsi-npg/npg_seq_pipeline/pull/119/files#r43224414>.
>
Yes, this is a strong argument. It's impossible now to single out npg 
scripts and sometimes I am thinking about renaming everything. M



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.